src/main/java/org/ohdsi/cohortcharacterization/utils/QuoteUtils.java

```diff
@@ -0,0 +1,16 @@
+package org.ohdsi.cohortcharacterization.utils;
```

Comment:
Should we consider placing `QuoteUtils` in `org.ohdsi.analysis` instead since it is re-used between this package and WebAPI?